crates/solidity/src/yul/parser/statement/variable_declaration.rs

```diff
@@ -101,7 +122,13 @@ where
     ) -> anyhow::Result<()> {
         if self.bindings.len() == 1 {
             let identifier = self.bindings.remove(0);
-            let r#type = identifier.r#type.unwrap_or_default().into_llvm(context);
+            if context.debug_info().is_some() {
+                let line_num: u32 =
```

Comment:
There are different integer sizes in use through the code. It seems better to be explicit about which size is needed here because it is being constructed to meet the inkwell function's requirements.

Reply:
Well, that'd basically require the receiving function to be generic over integer types, however, in this instance, it is used as a parameter to `fn set_debug_location` which requires the parameter to be `u32`. But as said it's fine for me either ways, no need to change it :)